src/pages/trademark.tsx

```diff
@@ -5,15 +5,18 @@ import Layout from '../components/Layout';
 import Article from '../components/Article';
 import Footer from '../components/Footer';
 import '../styles/article-reader.scss';
+import AboutPageSideNavBar from '../components/AboutPageSideNavBar';
 
 export default function TrademarkPage({ data }: Page): JSX.Element {
   const { title, description } = data.page.frontmatter;
   const { html, tableOfContents } = data.page;
   const { authors } = data.page.fields;
+
   return (
     <>
       <Layout title={title} description={description} showFooter={false}>
         <main className="centered-container">
+          <AboutPageSideNavBar pageTitle="Trademark Policy" />
```

Comment:
I think the component name should be more generic. The sidebar will be used on these pages

- about
- governance
- community
- working groups
- releases
- resources
- trademark
- and privacy policy

What is the pageTitle prop used for?

Reply:
It was used to identify the active page title. But, in the latest code, I introduced the `pageKey` property in navigation sections, and now we don't need the page title property instead we need to pass the `slug` as `pageKey` to identify the current viewing page.